### ISSUE TITLE
Add a whisper method

### ISSUE THREAD

**Issue by citycide** [d-fischer/twitch#62]
## Feature Request

Given other methods like `say` and `action` are implemented, the lack of a `whisper` method seems like an omission. Right now I believe you have to use the lower level `send`:

```ts
const whisper = (user: string, message: string) =>
  bot.send(`PRIVMSG #jtv :/w ${user} ${message}`)
```

That would round off the methods to send messages:

```ts
bot.say(channel, 'hello')
bot.action(channel, 'waves.')
bot.whisper(channel, 'hello again, but privately')
```

**Comment by d-fischer** [comment]
That definitely makes sense, however I probably won't get to it the next few days. If you absolutely can't wait, feel free to PR :)

**Comment by d-fischer** [comment]
Released with 3.5.0.
